platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -1650,7 +1664,7 @@ public void onSurfaceCreated(final GL10 gl, final EGLConfig config) {
     // ==================
 
     private void syncRemoveHttpHandle(final long requestHandle) {
```

Comment:
Not a big deal, but it might be clearer to just perform this synchronized remove in-line so that the calling code is more obvious.